config/locales/en.yml

```diff
@@ -967,6 +935,44 @@ toplevel:
         - type: text
           body: "<p>Get in touch with us at XXXX@biglotteryfund.org.uk</p>"
 about:
+  landing:
+    title: About the Big Lottery Fund
+    intro:
+      opening: We ensure money raised by players of <a href="https://www.national-lottery.co.uk/">The National Lottery</a> helps communities to achieve their ambitions and thrive. We do this by awarding grants, bringing communities together, and sharing what we learn with communities, other funders and the government.
+      learnApply: Find out how to apply for funding
+    whatWeDo:
+      title: What we do
+      paragraphs:
+      - <strong>The Big Lottery Fund funds projects and activities that make communities stronger and more vibrant, and that are led by the people who live in them.</strong>
+      - "We support charities, community groups, and people with great ideas - local or national, large or small.
+         We also bring people and groups together: to share experiences, learn from each other and try new ways of working."
+    howOurFundingIsUsed:
+      title: National Lottery funding
+      paragraphs:
+      - <strong>The National Lottery raises money for good causes.</strong>
+      - People use this funding to do extraordinary things,
+        taking the lead to improve their lives and communities.
+        Every time you buy a National Lottery ticket, you help make this happen.
+      - Communities come in all shapes and sizes, and National Lottery funding is there for everyone.
+      - There are 12 distributors of money raised by the players of The National Lottery, which fund projects and activities that transform communities, protect our heritage, and enriching lives through arts, sports and culture.
```

Comment:
"enriching" should be "enrich" (all the other verbs are present tense, eg. "fund", "transform", "protect" etc)